spec/components/schemas/dateperiod.yaml

```diff
@@ -8,5 +8,7 @@ properties:
         description: ISO 8601 date
         type: string
         format: date
+    reason:
```

Comment:
I'm unsure why we would want a 'reason' property inside a period?

Reply:
This is currently coming through in the API which is why this was added.
The reason field is used to give any termination notes when a period has a finish, I think it would make sense to include these when fetching someone's employment history

Reply:
Or would it make more sense for this to separated to its own 'employmentperiod' schema?